src/examples/README.md

```diff
@@ -14,6 +14,36 @@ can be implemented.
 In addition, here are some simple examples of utilizing the grpc_requests library
 in particular scenarios.
 
+## Interceptors
+
+Clients can be instantiated with interceptors to accomplish a variety of tasks,
+such authentication.
+
+Interceptors are provided to clients as an array of Interceptors when a client is instantiated.
+
+### Providing Authentication with a Metadata Interceptor
+
+```python
+from grpc_requests import Client
+
+class MetadataInterceptor(grpc.UnaryUnaryClientInterceptor):
```

Comment:
This class is arguably more of a `BearerTokenInterceptor` than a "generic" `MetadataInterceptor`, I think?

Reply:
Fair call out and better example - thank you!